### ISSUE TITLE
[Feature] Add option to disable update release page open

### ISSUE THREAD

**Issue by glottisfaun0000** [PedroS11/twitch-live-extension#66]
Add option to disable the release page opening when the extension updates. It's pretty obnoxious, thanks.

**Comment by PedroS11** [comment]
Heey @glottisfaun0000 are you having this issue? I haven't released anything in months, if so, I may add an option for it or completely disable them as they shouldn't be popping

**Comment by glottisfaun0000** [comment]
Yeah, everytime I start firefox up from scratch it opens the releases page.

**Comment by PedroS11** [comment]
Ohhh you are using firefox? that is an old version of the extension because firefox isn't support the new standards, I may need some time to have a look at that, sorry for the troubles!

**Comment by Makezu90** [comment]
also want add everytime firefox has update and need relaunch this release page opening, kinda annoying..

**Comment by PedroS11** [comment]
The feature was removed
